tests/01-sanity.sh

```diff
@@ -29,4 +29,18 @@ stest 1 sudo false
 stest 0 which singularity
 stest 0 test -f "$SINGULARITY_sysconfdir/singularity/singularity.conf"
 
+# Check whether singularity binary is on a volume mounted with 'nosuid'
+# I guess one could just grep /etc/mtab, but...
+mount | grep $(df -h $SINGULARITY_PATH | grep -v File | awk '{print $1}') \
```

Comment:
I would also be paranoid and do something like:

`mount | grep $(df -h $SINGULARITY_PATH | grep -v '^Filesystem' | awk '{print $1}')`

Just make sure the grep on the `df` output matches 'Filesystem' at the start of a line. You don't want to know some of the names I've seen mounted filesystems... Your grep would have ignored the mount point.